### ISSUE TITLE
Intermittent nature to breastfeeding

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10077]
**Preferred term label:**
Intermittent nature to breastfeeding

**Synonyms**

**Definition (free text, please give PubMed ID)**
The on and off character can predispose to Galactocele

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this a bit detailed given we have a term now for "History of recent breastfeeding"
Perhaps the latter term should be adjusted to include "intermittent"?

**Comment by MickeySegal** [comment]
It is the on and off character that is the risk factor for Galactocele.

Most women breastfeed regularly to feed the baby or pump if travelling.

**Comment by pnrobinson** [comment]
I adapted the term as follows

HP:6001045
label: History of recent intermittent breastfeeding
def: History of recently intermittent breastfeeding an infant (i.e., on and off).

**Comment by MickeySegal** [comment]
But we had used HP:6001045 for "Breast feeding ongoing in female" and now that is no longer the meaning.  See https://github.com/obophenotype/human-phenotype-ontology/issues/10076
